lib/run/index.js

```diff
@@ -340,10 +349,13 @@ module.exports = function (options, callback) {
                         });
                     });
 
-                    asyncEach(emitter.exports, exportFile, function (err) {
+                    asyncEach(emitter.exports, exportFile, async function (err) {
                         // we now trigger actual done event which we had overridden
                         emitter.emit('done', err, emitter.summary);
-                        callback(err, emitter.summary);
+
+                        var resultUploaded = await new uploadRunToPostman(uploadConfig, emitter.summary).start();
```

Comment:
This could be just a method call instead if we only ever need one method from the instance of the class. Why create a class for this at all?

Reply:
```suggestion
                        new uploadRunToPostman(uploadConfig, emitter.summary).start().then((resultUpdated) => 
```